Quaver.Shared/QuaverGame.cs

```diff
@@ -819,6 +847,36 @@ private void LimitFpsOnInactiveWindow()
             WindowActiveInPreviousFrame = IsActive;
         }
 
+        /// <summary>
+        ///     If the value for volume percentage scaling on focus loss is not set to maximum, checks if the window is unfocused and updates the unfocused property.
+        /// </summary>
+        private void CheckGameWindowFocus()
+        {
+            if (ConfigManager.VolumePercentageOnWindowInactive.Value == 100)
+                return;
+
+            WindowUnfocused = !IsActive;
+        }
+
+        /// <summary>
+        ///     Updates the game volume when focus changes.
+        /// </summary>
+        /// <param name="sender">The object which triggered the event.</param>
+        /// <param name="e">Extra event arguments.</param>
+        private void ScaleVolumeOnFocusChange(object sender, EventArgs e)
+        {
+            if (WindowUnfocused)
+            {
+                var newVolume = (int) (ConfigManager.VolumeGlobal.Value * (ConfigManager.VolumePercentageOnWindowInactive.Value / 100.0f));
+                OriginalVolume = ConfigManager.VolumeGlobal.Value;
+                ConfigManager.VolumeGlobal.Value = newVolume;
```

Comment:
Changing a user's config to update the volume is not a good solution.

Try changing `AudioTrack.GlobalVolume` and `AudioSample.GlobalVolume` instead. This way you won't overwrite the user's preferred setting (in the event of the game closing with the config being saved back to the original value).